html5lib/html5parser.py

```diff
@@ -1706,6 +1706,7 @@ def startTagColgroup(self, token):
 
         def startTagCol(self, token):
             self.startTagColgroup(impliedTagToken("colgroup", "StartTag"))
+            token["selfClosingAcknowledged"] = True
```

Comment:
Do we actually need this here before it gets returned and reprocessed? I'd like to think that the reprocessing handled this fine. (If it doesn't, it should be fixed!)